package.json

```diff
@@ -29,6 +29,7 @@
     "firebase-admin": "^11.11.1",
     "framer-motion": "^11.0.14",
     "lodash": "^4.17.21",
+    "markdown-to-jsx": "^7.4.6",
```

Comment:
are you still using markdown-to-jsx or react-markdown? if not could you uninstall deps pls